casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala

```diff
@@ -109,20 +110,15 @@ class InterpreterUtilTest
                 postGenStateHash,
                 postGenProcessedDeploys
               )
-          genPostStateT <- runtimeManager.storageRepr(postGenStateHash)
-          genPostState  = genPostStateT.get
-
-          _                                         = genPostState.contains("@{2}!(2)") should be(true)
-          _                                         = genPostState.contains("@{123}!(5)") should be(true)
+          _                                         <- getDataAtPublicChannel[Task](postGenStateHash, 2).map(_ shouldBe Seq("2"))
```

Comment:
Will this give a clean line number in the error message if it ever fails?

Reply:
Yes